localization/sparse_mapping/readme.md

```diff
@@ -50,46 +51,48 @@ Usually the bags are acquired at a very high frame rate, and they are
 huge. A preliminary filtering of the bag images while still on the
 robot can be done with the command:
 
-    rosbag filter input.bag output.bag "(topic == '/hw/cam_nav') and (float(t.nsecs)/1e+9 <= 0.1)"
+    rosbag filter input.bag output.bag                           \
+      "(topic == '/hw/cam_nav') and (float(t.nsecs)/1e+9 <= 0.1)"
 
 Here, for every second of recorded data, we keep only the first tenth
 of a second. This number may need to be adjusted. Later, a further
 selection of the images can be done.
 
-### Copy the bag from the robot:
+### Copy the bag from the robot
 
 From the local machine, fetch the bag:
 
     rsync -avzP astrobee@10.42.0.32:/data/bagfile.bag .
 
-Here, the IP address of P4D was used, which may differ from your robot's IP address.
+Here, the IP address of P4D was used, which may differ from your
+robot's IP address.
 
 ### Merging bags
 
 The bags created on the ISS are likely split into many smaller bags,
 for easy and reliability of transfer. Those can be merged into one bag
 as follows:
 
-    astrobee_build/devel/lib/localization_node/merge_bags \
+    astrobee/build/devel/lib/localization_node/merge_bags \
```

Comment:
should be astrobee/devel/lib/localization_node/merge_bags according to the default setup